sigs/micro/SIG-request.md

```diff
@@ -0,0 +1,41 @@
+# Request for SIG
+
+## What is this group for?
+
+For discussion and collaborations around running TensorFlow models on microcontrollers, DSPs, and other highly-resource constrained embedded devices. There are no hard rules about what counts as embedded, but this group concentrates on the challenges of running on devices that may only have hundreds or even tens of kilobytes of memory and storage.
+
+The TensorFlow Lite for Microcontrollers project was launched at the Dev Summit in March 2019 and has already attracted interest from many hardware and product partners outside of Google.
+
+## Who will be part of it?
+
+Describe:
+
+* Neil Tan from Arm will lead the group.
+* Pete Warden from Google will act as his second.
+* Multiple hardware and toolchain partners have expressed interest in joining the group.
+
+Membership will be entirely public.
+
+## What initial problems will the group tackle?
+
+Prioritize the roadmap for development.
+Divide up high priority work between contributors.
+Share support and help between contributors.
+
+## What modes of communication do you intend to use?
+
+We have a mailing list, will arrange a monthly VC call, and will be asking one of our partners to set up a Slack channel (since the process at Google is involved).
```

Comment:
For the other SIGs, we use Gitter, which has no setup process. e.g. see https://gitter.im/tensorflow/sig-build -- if there's no direct preference to use Slack, we would prefer Gitter. It has no commercial limitations on archiving, etc, and only needs a Github logon, vs creating an admin overhead.

Reply:
Thanks, that's helpful, I've set up a chat room at https://gitter.im/tensorflow/sig-micro and updated the proposal.